manager/handlers/has2be/sign_certificate.go

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package has2be
+
+import (
+	"context"
+	handlers201 "github.com/thoughtworks/maeve-csms/manager/handlers/ocpp201"
+	"github.com/thoughtworks/maeve-csms/manager/ocpp"
+	typesHasToBe "github.com/thoughtworks/maeve-csms/manager/ocpp/has2be"
+	types201 "github.com/thoughtworks/maeve-csms/manager/ocpp/ocpp201"
+)
+
+type SignCertificateHandler struct {
+	Handler201 handlers201.SignCertificateHandler
+}
+
+func (s SignCertificateHandler) HandleCall(ctx context.Context, chargeStationId string, request ocpp.Request) (ocpp.Response, error) {
+	req := request.(*typesHasToBe.SignCertificateRequestJson)
```

Comment:
The v2.0.1 handler expects the CSR to be PEM encoded. Some (and only some) charge stations using the Has2Be extensions instead encode them as base64 DER. We need to see if there is a PEM block (using pem.Decode) and if we don't get a PEM block back then we need to base64 decode the CSR and then reformat as a PEM CSR using pem.EncodeToString before passing to the v2.0.1 handler

Reply:
Also please make sure we record in our open telemetry instrumentation what the input format was.